### ISSUE TITLE
DFlash compact verify scores logits against a requantized LM head instead of the native Q4_K

### ISSUE THREAD

**Issue by JSONbored** [gittensor-ai-lab/sparkinfer#719]
## Summary

`dflash_verify_short_run` scores its logits against an int8 requantization of the Q4_K LM head,
while autoregressive decode scores against the native Q4_K weights. The two paths therefore compute
logits from different weight representations.

## Detail

The int8 copy is built by `launch_gguf_dequant_rows_i8`, which is symmetric int8 with a single
scale per row:

```
q[r,c] = round(v[r,c] / scale[r]),   scale[r] = max_c|v[r,c]| / 127
```

That is one scale for all 2048 columns of a row, against Q4_K's per-32-weight block scales plus
6-bit sub-scales across the same row. Every row of every verify batch therefore carries a
systematic logit error that AR never sees.

It is also not a bandwidth win: at `vocab=248320 x H=2048` the int8 head is 508 MB against the
Q4_K head's ~286 MB, so the requantized path reads ~78% more per verify call.

## Fix

`launch_mmvq_rows_f32` already supports `qtype=12` at `K=2048` via `si_mmvq_q4k_rows_exact_kernel`,
so the batched path can read the same native head AR uses without a new kernel.
